remix-tests/src/index.js

```diff
@@ -164,8 +164,26 @@ var runTestFiles = function (filepath, isDirectory, web3, opts) {
       }
 
       if (isDirectory) {
-        fs.readdirSync(filepath).forEach(filename => {
-          gatherContractsFrom(filename)
+        fs.walkSync = function (start, callback) {
```

Comment:
same `fs.walkSync` is used in compiler.js. could we put that in an helper class?

Reply:
Please see #1063